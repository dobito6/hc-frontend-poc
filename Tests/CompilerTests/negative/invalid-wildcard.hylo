//! no-std

fun main() { _ }
