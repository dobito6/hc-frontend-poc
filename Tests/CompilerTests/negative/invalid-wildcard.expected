negative/invalid-wildcard.hylo:1.15: error: expected '='
fun main() { _ }
              ^
